EdvClient.js

```diff
@@ -466,9 +466,6 @@ export class EdvClient {
     const self = this;
     const {cipher} = this;
     const state = doc.stream;
-    if(!state) {
```

Comment:
Since there is a non-optional assert check for the stream before this, I don't think we will ever get to this line.

Reply:
Yes, I think this is safe to remove.

Reply:
This looks fine.